which_language.md

```diff
@@ -190,3 +190,46 @@ Materials
 - [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS)
 - [JS for C & Python programmers](https://www.wooji-juice.com/blog/javascript-article.html)
 
+Lua
+==========
+A light, somewhat verbose Python-like language where indentation doesn't matter and almost everything is a table. Also, indexing starts from one. Typically, the Lua VM sits on top of a lower-level engine, providing an easy-to-use scripting interface.
+
+Usual usage
+-----------
+- video game scripting & modding
+- game development (Love2D, Pico-8, TIC-80)
+  
+Pros
+----
+
+- tiny & portable
+- short learning curve
+- fast, especially when using the JIT compiler
```

Comment:
A bit questionable/misleading to call Lua fast, when there are things like C, C++, Java and C# in the same list.

Also, again.  Do not assume reader knows what a "JIT compiler" is.